vendor/github.com/containernetworking/cni/pkg/types/current/types.go

```diff
@@ -22,7 +22,7 @@ import (
 	"os"
 
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/containernetworking/cni/pkg/types/020"
+	types020 "github.com/containernetworking/cni/pkg/types/020"
```

Comment:
Hey Nikhil, is this necessary? In theory, we should be able to keep the vendored packages the same.